### ISSUE TITLE
Download button should have white border

### ISSUE THREAD

**Issue by arelsirin** [arelsirin/open-learn#3]
<img width="782" alt="screen shot 2016-08-05 at 3 30 05 pm" src="https://cloud.githubusercontent.com/assets/2131732/17449847/bddaf156-5b22-11e6-9886-97e1cad102ce.png">

**Comment by ismi2014** [comment]
Should the border be white when hovering?

**Comment by arelsirin** [comment]
Yes, white on active, focus and hover, and its width must be 2px.

**Comment by raiman264** [comment]
hey @ismi2014 your change also affects the "SignUp" button at bottom

@arelsirin use my fixe

**Comment by ismi2014** [comment]
Hey raiman264, 

Changes should be consistent and affect all buttons.

**Comment by raiman264** [comment]
who said that?
